llvm/lib/Target/RISCV/RISCVRegisterInfo.cpp

```diff
@@ -823,6 +823,8 @@ bool RISCVRegisterInfo::getRegAllocationHints(
         tryAddHint(MO, MI.getOperand(0), NeedGPRC);
       }
     }
+    if (MI.isFusible() && OpIdx == 1)
```

Comment:
don't we need to check that the "first" instuction is part of a fusible pair?

Reply:
I think we don't need to, because this has been done in MacroFusion mutation?